js_modules/dagster-ui/packages/ui-core/src/assets/__tests__/AssetCatalogTablev2.test.tsx

```diff
@@ -195,69 +192,75 @@ describe('AssetCatalogTableV2', () => {
       ),
     );
 
-    await waitFor(() => {
-      expect(calls[calls.length - 1][0]).toEqual({
-        groupedByStatus: {
-          Healthy: [
-            expect.objectContaining({key: buildAssetKey({path: ['asset1']})}),
-            expect.objectContaining({key: buildAssetKey({path: ['asset5']})}),
-          ],
-          Degraded: [expect.objectContaining({key: buildAssetKey({path: ['asset2']})})],
-          Warning: [expect.objectContaining({key: buildAssetKey({path: ['asset3']})})],
-          Unknown: [expect.objectContaining({key: buildAssetKey({path: ['asset4']})})],
-        },
-        loading: false,
-        healthDataLoading: false,
-      });
-    });
+    const props = calls[calls.length - 1][0];
+    expect(props.checkedDisplayKeys).toEqual(new Set());
+    expect(props.loading).toBe(false);
+    expect(props.healthDataLoading).toBe(false);
+
+    expect(props.groupedByStatus.Healthy).toHaveLength(2);
+    const [healthy1, healthy2] = props.groupedByStatus.Healthy;
+    expect(healthy1.key).toEqual(buildAssetKey({path: ['asset1']}));
+    expect(healthy2.key).toEqual(buildAssetKey({path: ['asset5']}));
+
+    expect(props.groupedByStatus.Degraded).toHaveLength(1);
+    const [degraded] = props.groupedByStatus.Degraded;
+    expect(degraded.key).toEqual(buildAssetKey({path: ['asset2']}));
+
+    expect(props.groupedByStatus.Warning).toHaveLength(1);
+    const [warning] = props.groupedByStatus.Warning;
+    expect(warning.key).toEqual(buildAssetKey({path: ['asset3']}));
+
+    expect(props.groupedByStatus.Unknown).toHaveLength(1);
+    const [unknown] = props.groupedByStatus.Unknown;
+    expect(unknown.key).toEqual(buildAssetKey({path: ['asset4']}));
   });
```

Comment:
I find breaking it apart into a bunch of smaller checks creates more mental overhead. Maybe we could simplify the previous logic a little to remove the boilerplate instead?


```
const toKeyList = (paths) => paths.map(key => expect.objectContaining({key: buildAssetKey({path)}));
await waitFor(() => {
      expect(calls[calls.length - 1][0]).toEqual({
        groupedByStatus: {
          Healthy: toKeyList([['asset1'], ['asset5']]),
          Degraded: toKeyList(['asset2']),
          Warning: toKeyList(['asset3']),
          Unknown: toKeyList(['asset4']),
        },
        loading: false,
        healthDataLoading: false,
      });
    });
```